src/MSBuild/XMake.cs

```diff
@@ -2428,7 +2433,7 @@ out enableProfiler
                         Console.WriteLine(ResourceUtilities.FormatResourceStringStripCodeAndKeyword("PickedUpSwitchesFromAutoResponse", autoResponseFileName));
                     }
 
-                    if (verbosity == LoggerVerbosity.Diagnostic)
+                    if (verbosity == LoggerVerbosity.Diagnostic && originalVerbosity != LoggerVerbosity.Quiet)
```

Comment:
Maybe just:
```suggestion
                    if (originalVerbosity == LoggerVerbosity.Diagnostic)
```

?

Reply:
when bl is set and other verbosity than quiet/diagnostic is set, the output is still wanted.

Reply:
This is interesting. My instinct was to suppress the message for "quiet", "minimal" and "normal" but leave it for "detailed" and "diagnostic". Why "quiet" only?

Reply:
Hmm... we need more requirement clarifications. My understanding of the requirement is solely from the #7828 statement. It seems we three have different understandings, let me know how we can make an agreement. @rainersigwald  / @Forgind

Reply:
@Forgind and I actually talked about this this morning, and I think it's ok for it to be there for "diagnostic" only.